extra/2-piping.js

```diff
@@ -16,27 +16,28 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(a, b) {
+  return a + b;
 }
 
-function multiply() {
-
+function multiply(a, b) {
+  return a * b;
 }
 
-function format() {
-
+function format(number) {
+  return `£${number.toString()}`;
 }
 
 const startingValue = 2;
 
 // Why can this code be seen as bad practice? Comment your answer.
```

Comment:
Still need to answer this question (with a comment)

Reply:
Sorry, I forgot about this.